modules/otel-config-multicluster.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+// 
+// * otel/otel-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-observability-data-from-different-clusters_{context}"]
+= Gathering the observability data from different clusters with the OpenTelemetry Collector
+
+For a multicluster configuration, you can create one OpenTelemetry
+Collector instance in each one of the remote clusters and forward all the telemetry
+data to one OpenTelemetry Collector instance.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {TempoOperator} is installed.
+* A TempoStack is deployed on the cluster.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.Example ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+----
+
+. Create a cluster role for the service account.
++
+.Example ClusterRole
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  <1>
+  <2>
+- apiGroups: ["", "config.openshift.io"]
+  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+----
+<1> The `k8sattributesprocessor` requires permissions for pods and namespace resources.
+<2> The `resourcedetectionprocessor` requires permissions for infrastructures and status.
+
+. Bind the cluster role to the service account.
++
+.Example ClusterRoleBinding
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-deployment
+  namespace: otel-collector-<example>
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+----
+
+. Create the YAML file to define the `OpenTelemetryCollector` custom resource (CR) in the edge clusters.
++
+.OpenTelemetryCollector for the edge clusters
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: otel-collector-<example>
+spec:
+  mode: daemonset
+  serviceAccount: otel-collector-deployment
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      opencensus:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      zipkin:
+    processors:
+      batch:
+      k8sattributes:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 50
+        spike_limit_percentage: 30
+      resourcedetection:
+        detectors: [openshift]
+    exporters:
+      otlphttp:
+        endpoint: https://observability-cluster.com:443 <1>
+        insecure: false
+        compression: on
+        tls:
```

Comment:
You need to generate them in the central one

Reply:
And how are they used in the central one? This is missing in the docs

Reply:
I created them using this:
```yaml
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: letsencrypt-staging
  namespace: observability
spec:
  acme:
    preferredChain: ""
    privateKeySecretRef:
      name: private-key-lets-encrypt
    server: https://acme-staging-v02.api.letsencrypt.org/directory
    solvers:
    - http01:
        ingress:
          class: openshift-default
```

I just checked my notes. I found why I added the annotation and was because I wrote this following the steps from this blogpost: https://cloud.redhat.com/blog/observability-across-openshift-cluster-boundaries-with-distributed-data-collection

I ran the thing in my cluster and I was not sure if we wanted to add the cert generation to the documents or just let the customer decide how to proceed with this. I didn't add the cert manager object but I forget to remove the annotation.